utils/vscode/development.md

```diff
@@ -14,6 +14,12 @@ NodeJS is required to build the extension. You will also need to install `vsce`:
 npm install -g vsce
 ```
 
+Add vsce to your path:
+
+```
+export PATH=$PATH:/usr/local/lib/node_modules/vsce
```

Comment:
Did you do `-g`? That should have added it to something like /usr/local/bin...

```
$ which vsce
/usr/local/bin/vsce
```

Reply:
Looks like I don't have /usr/local/bin in my path.

Reply:
PTAL

Reply:
Maybe advise /usr/local/bin here instead, then? Or, if you prefer, `$(npm config get prefix)/bin`

Note I'm a little surprised you don't have /usr/local/bin -- I'd have thought we'd be using similar user environments, and it's in my $PATH on both mac and linux.

Reply:
Ah, I see the change now. Note, still not sure why you don't have /usr/local/bin by default, but I don't think that really affects this suggestion.